discourse_rock/patches/db_migrations.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/db/post_migrate/20240212034010_drop_deprecated_columns.rb b/db/post_migrate/20240212034010_drop_deprecated_columns.rb
```

Comment:
Have you contributed this change upstream too?

Reply:
Yes, please :) https://github.com/discourse/discourse

Reply:
Sure :D but I think my fix is pretty janky :sweat_smile:

Reply:
:D Nah, I think it's fine :)